docs/OV_Runtime_UG/model_representation.md

```diff
@@ -10,7 +10,8 @@ Each operation in `ov::Model` has the `std::shared_ptr<ov::Node>` type.
 
 For details on how to build a model in OpenVINO™ Runtime, see the [Build a Model in OpenVINO™ Runtime](@ref build_model) section.
 
-OpenVINO™ Runtime allows using tensor names or indexes to work wit model inputs/outputs. To get model input/output ports, use the `ov::Model::inputs()` or `ov::Model::outputs()` methods respectively.
+OpenVINO™ Runtime allows using tensor names or indexes to work with model inputs/outputs, so it means that you don't use OpenVINO specific conventions to operate with model inputs/outputs anymore, you use native framework names or indexes to work with them. 
```

Comment:
Here there is no clear understanding what `OpenVINO specific conventions` mean so this opposition doesn't give desired learning effect. It is better to describe the flow in more imperative terms:

1. Take a name from a framework model (you need to use tensor names conventions specific for frameworks, provide a link to per framework description -- needs collaboration with MO guys to build a correct link as they together with FEs own naming mapping rules).
2. Provide a name to OV API to query corresponding input/output
3. Avoid iterating over names in OV model and map name back to framework as this may result into multiple names attached to the same node (describe shortly why).

@mvafin please suggest links to model importing documentation to support this story.

Reply:
@slyalin Changed this text.
MO uses operation names and I didn't find the place in the documentation where we have information about conventions.